superset/config.py

```diff
@@ -359,6 +359,7 @@ def _try_json_readsha(  # pylint: disable=unused-argument
     "OMNIBAR": False,
     "DASHBOARD_RBAC": False,
     "ENABLE_EXPLORE_DRAG_AND_DROP": False,
+    "ALERTS_ATTACH_REPORTS": True,
```

Comment:
nit: would be nice to have a small explanation here

Reply:
another nit, does it make sense for this to be a feature flag? I mean `ALERT_REPORTS` is a feature flag, with lot's of other configuration options that live outside the feature flag dict. Also it may be more clear to call this flag `ALERTS_ATTACH_SCREENSHOT`